cypress/README.md

```diff
@@ -8,7 +8,7 @@ The tests are automatically run by the CI environment.
 
 To run them locally, you can use the Cypress test runner. Make sure you have the correct dependencies installed by running: `yarn cypress verify`
 
-If you want to, you can override the launch URLs:  
+If you want to, you can override the launch URLs:
```

Comment:
You need to disable trailing whitespace trimming for Markdown. :D 

In VS Code I use:

```json
"[markdown]": {
    "files.trimTrailingWhitespace": false,
    "editor.formatOnSave": false
}
```